### ISSUE TITLE
[SQL] missing Postgres data types

### ISSUE THREAD

**Issue by basaran** [sublimehq/Packages#2899]
- Sublime Version: 3.2
- OS Version: Linux

```sql
CREATE TABLE IF NOT EXISTS public.dropzone_details
(
    id uuid NOT NULL DEFAULT uuid_generate_v4(),
    name text COLLATE pg_catalog."default",
    doc jsonb,
    poc json,
    size integer,
    CONSTRAINT id_name_uix UNIQUE (id, name)
)
```

Postgresql has some new data types introduced since version 9.5. It appears some of them are missing in `SQL.sublime-syntax`.

These data types should be added to `SQL.sublime-syntax`:

uuid
jsonb
json
xml
pg_snapshot
pg_lsn
txid_snapshot

Reference: https://www.postgresql.org/docs/13/datatype.html

**Comment by michaelblyons** [comment]
Postgres isn't (yet) a part of #3046. Someone could add an extra extended file and start adding one, though. A handful of data types would be an easy place to start.

**Comment by jrappen** [comment]
From my understanding @keith-hall is still working on it, so now would be a good time to add this to the TODOs there I guess.

**Comment by jrappen** [comment]
Partly addressed by #3046 now.

**Comment by deathaxe** [comment]
Note: Postgre is still incomplete right after merging PR #3046.
